static/app/views/replays/list/replaySearchAlert.tsx

```diff
@@ -0,0 +1,107 @@
+import useProjects from 'sentry/utils/useProjects';
+import withSdkUpdates from 'sentry/utils/withSdkUpdates';
+
+interface ReplaySearchAlertProps {
+  sdkUpdates?: ProjectSdkUpdates[] | null;
+}
+
+const MIN_REPLAY_CLICK_SDK = '7.44.0';
+
+// exported for testing
+export function ReplaySearchMinSdkAlert({sdkUpdates}: ReplaySearchAlertProps) {
+  const {selection} = usePageFilters();
+  const projects = useProjects();
+  const location = useLocation();
+  const conditions = useMemo(() => {
+    return new MutableSearch(decodeScalar(location.query.query, ''));
+  }, [location.query]);
+
+  const hasReplayClick = conditions.getFilterKeys().some(k => k.includes('click'));
+
+  if (!hasReplayClick) {
+    return null;
+  }
+
+  const selectedProjectsWithSdkUpdates = sdkUpdates?.reduce((acc, sdkUpdate) => {
+    if (!selection.projects.includes(Number(sdkUpdate.projectId))) {
+      return acc;
+    }
+
+    const project = projects.projects.find(p => p.id === sdkUpdate.projectId);
+    // should never really happen but making ts happy
+    if (!project) {
+      return acc;
+    }
+
+    acc.push({
+      project,
+      sdkUpdate,
+    });
+
+    return acc;
+  }, [] as Array<{project: Project; sdkUpdate: ProjectSdkUpdates}>);
+
+  const doesNotMeetMinSDK =
+    selectedProjectsWithSdkUpdates &&
+    selectedProjectsWithSdkUpdates.length > 0 &&
+    selectedProjectsWithSdkUpdates.every(({sdkUpdate}) => {
+      return semverCompare(sdkUpdate.sdkVersion, MIN_REPLAY_CLICK_SDK) === -1;
+    });
+
+  if (!doesNotMeetMinSDK) {
+    return null;
+  }
+
+  const sdkUpdateAction = selectedProjectsWithSdkUpdates?.[0]?.sdkUpdate.suggestions.find(
+    suggestion => suggestion.type === 'updateSdk'
+  ) as UpdateSdkSuggestion | undefined;
+
+  if (sdkUpdateAction) {
+    return (
+      <Alert data-test-id="min-sdk-alert">
+        {tct(
+          'Searching by click requires a minimum SDK version of [sdkName]@v[minSdkVersion]. [action]',
+          {
+            sdkName: sdkUpdateAction.sdkName,
+            newSdkVersion: sdkUpdateAction.newSdkVersion,
+            minSdkVersion: MIN_REPLAY_CLICK_SDK,
+            action: sdkUpdateAction.sdkUrl ? (
+              <ExternalLink href={sdkUpdateAction.sdkUrl}>
+                {tct('Update to [sdkName]@v[newSdkVersion]', {
+                  sdkName: sdkUpdateAction.sdkName,
+                  newSdkVersion: sdkUpdateAction.newSdkVersion,
+                })}
+              </ExternalLink>
+            ) : (
+              tct('Update to [sdkName]@v[newSdkVersion]', {
+                sdkName: sdkUpdateAction.sdkName,
+                newSdkVersion: sdkUpdateAction.newSdkVersion,
+              })
+            ),
+          }
+        )}
+      </Alert>
+    );
+  }
+
+  return (
+    <Alert data-test-id="min-sdk-alert">
+      {tct('Searching by click requires a minimum SDK version of v[minSdkVersion].', {
+        minSdkVersion: MIN_REPLAY_CLICK_SDK,
+      })}
+    </Alert>
+  );
```

Comment:
all of this can be moved to another component, and branching logic can exist here to support other cases where we need to alert for a "min sdk"